### ISSUE TITLE
[PyTorch] Add torch.compile() for frequently-used models

### ISSUE THREAD

**Issue by mikesvoboda** [mikesvoboda/nemotron-v3-home-security-intelligence#3880]
## Summary

Add torch.compile() to RT-DETRv2, Florence-2, CLIP for 15-40% latency reduction.

## Implementation

After model.eval(), add:
model = torch.compile(model, mode='reduce-overhead')

## Expected Benefit

* 15-40% latency after warmup
* Better GPU utilization

## Labels: optimization, ai, pytorch, phase-2
